lib/samples/find_address_with_reverse_geocode/find_address_with_reverse_geocode_sample.dart

```diff
@@ -46,159 +49,73 @@ class _FindAddressWithReverseGeocodeSampleState
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      key: _scaffoldStateKey,
-      endDrawer: Drawer(
-        child: SafeArea(
-          child: Padding(
-            padding: const EdgeInsets.all(10),
-            child: buildServiceControls(context),
-          ),
-        ),
-      ),
-      resizeToAvoidBottomInset: false,
       body: Stack(
         children: [
           ArcGISMapView(
             controllerProvider: () => _mapViewController,
             onMapViewReady: onMapViewReady,
-            onTap: onTap,
-          ),
-          Visibility(
-            visible: _ready,
-            child: SafeArea(
-              child: Container(
-                margin: const EdgeInsets.all(10),
-                child: buildSearchControls(context),
-              ),
-            ),
+            onTap: _ready ? onTap : null,
           ),
         ],
       ),
     );
   }
 
-  Widget buildServiceControls(BuildContext context) {
-    return Column(children: [
-      const Text('Geocode Service'),
-      RadioListTile<LocatorTask>(
-        title: const Text('World'),
-        value: _worldLocatorTask,
-        groupValue: _locatorTask,
-        onChanged: onServiceChanged,
-      ),
-      RadioListTile<LocatorTask>(
-        title: const Text('San Diego'),
-        value: _sanDiegoLocatorTask,
-        groupValue: _locatorTask,
-        onChanged: onServiceChanged,
-      ),
-    ]);
-  }
-
-  Widget buildSearchControls(BuildContext context) {
-    return Row(
-      children: [
-        Expanded(
-          child: Container(
-            padding: const EdgeInsets.symmetric(horizontal: 10),
-            color: Colors.white,
-            child: TextField(
-              decoration: const InputDecoration(
-                hintText: 'Search...',
-              ),
-              onSubmitted: onSearchSubmitted,
-            ),
-          ),
-        ),
-        IconButton(
-          icon: const Icon(
-            Icons.list,
-            color: Colors.white,
-          ),
-          onPressed: () => _scaffoldStateKey.currentState!.openEndDrawer(),
-        ),
-      ],
-    );
-  }
-
   void onMapViewReady() async {
-    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISImagery);
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
 
     final image = await ArcGISImage.fromAsset('assets/pin_circle_red.png');
-    final pictureMarkerSymbol = PictureMarkerSymbol.withImage(image);
-    pictureMarkerSymbol.width = 35;
-    pictureMarkerSymbol.height = 35;
-    pictureMarkerSymbol.offsetY = pictureMarkerSymbol.height / 2;
+    final pictureMarkerSymbol = PictureMarkerSymbol.withImage(image)
+    ..width = 35
+    ..height = 35
+    ..offsetY = pictureMarkerSymbol.height / 2;
```

Comment:
@jenmerritt line 71 is giving error because pictureMarkerSymbol is being referenced before it is done being declared - is it okay if reset it to what it was (line 131)

Reply:
Ohh yeah, yeah I'd just do that line separately / revert that bit to what you had 👍